backend/routes.js

```diff
@@ -236,6 +254,18 @@ route.GET('/eventsAfter/{contractID}/{since}/{limit?}', {}, async function (requ
   }
 })
 
+route.GET('/ownResources', {
+  auth: {
+    strategies: ['chel-shelter'],
+    mode: 'required'
+  }
+}, async function (request, h) {
+  const billableContractID = request.auth.credentials.billableContractID
+  const resources = (await sbp('chelonia.db/get', `_private_resources_${billableContractID}`))?.split('\x00')
+
+  return resources || []
+})
```

Comment:
Could you add a comment above line 257 explaining who is allowed to call this and an example of what this route is useful for?

While trying to understand `chel-shelter` authorization I noticed that `verifyShelterAuthorizationHeader` has:

```
 if (Math.abs(parseInt(timestamp) - (Date.now() / 1e3 | 0)) > 60) 
```

So it seems like `master` hasn't been merged here or that was missed.

Reply:
`master` hadn't been merged. An error like that would very likely result failing tests (due to being unable to create contracts).